src/esportsbot/bot.py

```diff
@@ -207,14 +207,24 @@ async def on_message(message):
     if not message.author.bot:
         # Ignore self messages
         guild_id = message.guild.id
-        music_channel_in_db = client.MUSIC_CHANNELS.get(guild_id)
-        if music_channel_in_db is not None and message.channel.id == music_channel_in_db:
+        if message.channel.id == client.MUSIC_CHANNELS.get(guild_id):
             # The message was in a music channel and a song should be found
             music_cog_instance = client.cogs.get('MusicCog')
             await music_cog_instance.on_message_handle(message)
             await client.process_commands(message)
             await message.delete()
 
+        elif message.guild is not None and message.role_mentions:
```

Comment:
This means that roles can be ping spammed in the music channel without triggering cooldown
Please see my `on_message` implementation i provided [in discord](https://discord.com/channels/213011069671571456/818102325172109332/841002357097496608)